assembl/static2/js/app/utils/timeline.js

```diff
@@ -163,4 +165,14 @@ export const getBarPercent = (_phase: Phase) => {
     }
   }
   return barWidth;
+};
+
+export const getPhaseId = (_timeline: Timeline, _identifier: string) => {
```

Comment:
`_identifier` is useless, you can use `identifier` directly for the param name.
You could probably use a default parameter value for timeline https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/default_parameters